sktime/forecasting/ttm.py

```diff
@@ -35,6 +35,28 @@ class TinyTimeMixerForecaster(_BaseGlobalForecaster):
     open-sourced by IBM Research. With less than 1 Million parameters, TTM introduces
     the notion of the first-ever "tiny" pre-trained models for Time-Series Forecasting.
 
+    **Zero-shot and Fine-tuning**:
+    The Model serves as a foundation model capable of both zero-shot forecasting and
+    fine-tuning on custom datasets. The decision to fine-tune is handled internally and
+    is determined by the configuration specified during model loading.
+
+    - **Zero-Shot Forecasting**: When the default configuration remains unchanged,
+      i.e ``config=None`` during model initialization,
+      the model weights are loaded as is, enabling zero-shot forecasting.
+
+    - **Fine-Tuning**: When the user provides a config different from default,
+      for example: ``config={"prediction_length": 2}`` during model initialization,
+      certain model weights are re-initialized to align with the new settings.
```

Comment:
I suppose can be more explicit, since only the weights are reinitialized that belong to a missing key.